HW05A_script.R

```diff
@@ -0,0 +1,20 @@
+#HW05 Part A
+#Author: Hannah Farrell
+#Date: 8/3/2020
+
+#load appropriate packages
+library(tidyverse)
+
+#load in the data
+dad_mom <- read.table("dad_mom.txt", header = TRUE)
+
```

Comment:
I think this was a very smart way to tidy the table. I was not familiar with the "gather" function. Definitely less convoluted to what I did (using pivot_longer, replace, mutate and select). I was able to get the table dad_mom_tidy and looks good. There is one warning saying "attributes are not identical across measure variables;
they will be dropped" but I don't think there is any problems.